### PR TITLE
created Twitter widget

### PR DESCRIPTION
**Pull request by rithikha**
Created (twitter widget) #340

### CODE REVIEW

File: ubyssey/static/src/styles/modules/widgets/_facebook_video_big.scss [hunk 1]
```diff
@@ -15,6 +15,29 @@
 	}
 }
 
+.c-facebook-video__title-container {
+
+	display: flex;
+  font-weight: 800;
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/modules/widgets/_facebook_video_big.scss]: Missing spacing here

File: ubyssey/widgets.py [hunk 2]
```diff
@@ -115,6 +115,43 @@ def context(self, result):
         return result
 
 @register.widget
+<<<<<<< Updated upstream
```
Comment (jumbosushi) [inline ubyssey/widgets.py]: Still have git merge leftovers in this file haha

File: ubyssey/static/src/styles/modules/widgets/_twitter.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.c-twitter-widget-container {
+height: 30rem;
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/modules/widgets/_twitter.scss]: Need spacing here

File: ubyssey/static/src/styles/modules/widgets/_facebook_video_big.scss [hunk 4]
```diff
@@ -15,6 +15,29 @@
...
+	margin-bottom: 1rem;
+	padding-left: 1rem;
+	padding-right: 1rem;
+
+
+
+	.c-facebook-video__title {
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/modules/widgets/_facebook_video_big.scss]: Spacing is bit different from others in here

**Review by jumbosushi — COMMENTED** [review]
Great start! There's a django bug when I run the site with this which I'll look into, but let's try to fix these styles first

File: ubyssey/widgets.py (hunk 2 above)
Comment (psiemens) [reply ubyssey/widgets.py]: This could be what's causing the server to crash

File: ubyssey/templates/widgets/frontpage/twitter-front-page.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div class="c-twitter-widget-container">
+<a class="twitter-timeline" href="https://twitter.com/ubyssey"></a>
```
Comment (psiemens) [inline ubyssey/templates/widgets/frontpage/twitter-front-page.html:2]: This shouldn't be hardcoded -- you can return the `{{ handle }}` value that is saved by the widget.

**Review by psiemens — CHANGES_REQUESTED** [review]

**Pull request merged**